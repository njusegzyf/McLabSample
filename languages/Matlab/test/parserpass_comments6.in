%{
Comment1
%}
a = 1 + 5;
b = 1 - 1;
%{
Comment2
%}