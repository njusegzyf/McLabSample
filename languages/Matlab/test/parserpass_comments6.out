>>>> Destination Language -> Source Language
[1, 1] -> (1, 1)
[1, 2] -> (1, 2)
[1, 3] -> (1, 3)
[2, 1] -> (2, 1)
[2, 2] -> (2, 2)
[2, 3] -> (2, 3)
[2, 4] -> (2, 4)
[2, 5] -> (2, 5)
[2, 6] -> (2, 6)
[2, 7] -> (2, 7)
[2, 8] -> (2, 8)
[2, 9] -> (2, 9)
[3, 1] -> (3, 1)
[3, 2] -> (3, 2)
[3, 3] -> (3, 3)
[4, 1] -> (4, 1)
[4, 2] -> (4, 2)
[4, 3] -> (4, 3)
[4, 4] -> (4, 4)
[4, 5] -> (4, 5)
[4, 6] -> (4, 6)
[4, 7] -> (4, 7)
[4, 8] -> (4, 8)
[4, 9] -> (4, 9)
[4, 10] -> (4, 10)
[4, 11] -> (4, 11)
[5, 1] -> (5, 1)
[5, 2] -> (5, 2)
[5, 3] -> (5, 3)
[5, 4] -> (5, 4)
[5, 5] -> (5, 5)
[5, 6] -> (5, 6)
[5, 7] -> (5, 7)
[5, 8] -> (5, 8)
[5, 9] -> (5, 9)
[5, 10] -> (5, 10)
[5, 11] -> (5, 11)
[6, 1] -> (6, 1)
[6, 2] -> (6, 2)
[6, 3] -> (6, 3)
[7, 1] -> (7, 1)
[7, 2] -> (7, 2)
[7, 3] -> (7, 3)
[7, 4] -> (7, 4)
[7, 5] -> (7, 5)
[7, 6] -> (7, 6)
[7, 7] -> (7, 7)
[7, 8] -> (7, 8)
[7, 9] -> (7, 9)
[8, 1] -> (8, 1)
[8, 2] -> (8, 2)
>>>> Translated Text
%{
Comment1
%}
a = 1 + 5;
b = 1 - 1;
%{
Comment2
%}